<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow:
{
background-color:rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget_invites">
      <property name="styleSheet">
       <string notr="true">QTabWidget:pane {
border:4px;
background-color:rgb(90, 90, 90);


}

QTabBar::tab:selected {
color:  rgb(255, 255, 255);
background-color: rgb(90, 90, 90);
}

QTabBar::tab:!selected {
color: rgb(255,255,255);
background-color:  rgb(107, 107, 107);

}


</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Invités</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string notr="true">color:rgb(255,255,255);</string>
            </property>
            <property name="title">
             <string>Ajouter Invité</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="1">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="cINLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>CIN</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="cINLineEdit">
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{

background-color: rgb(255,255,255);
color:tgb(0,0,0);

}</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="nomLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>Nom</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="nomLineEdit">
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{

background-color: rgb(255,255,255);

color:tgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="prNomLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>Prénom</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="prNomLineEdit">
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{

background-color: rgb(255,255,255);
color:tgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="dateDeNaissanceLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>Date de Naissance</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QDateEdit" name="dateDeNaissanceDateEdit">
                   <property name="styleSheet">
                    <string notr="true">QDateEdit
{

background-color: rgb(255,255,255);
color:tgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="eMailLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>E-mail</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QLineEdit" name="eMailLineEdit">
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{

background-color: rgb(255,255,255);
color:tgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="QLabel" name="sexeLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255,255,255);</string>
                   </property>
                   <property name="text">
                    <string>Sexe</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QComboBox" name="sexeComboBox">
                   <property name="styleSheet">
                    <string notr="true">QComboBox
{

background-color: rgb(0,0,0);

}</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Homme</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Femme</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="styleSheet">
                  <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                 </property>
                 <property name="text">
                  <string>Contract</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_reset">
                 <property name="styleSheet">
                  <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                 </property>
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_ajouter">
                 <property name="styleSheet">
                  <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                 </property>
                 <property name="text">
                  <string>Ajouter</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="styleSheet">
             <string notr="true">color:rgb(255,255,255);</string>
            </property>
            <property name="title">
             <string>Afficher Invités</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLineEdit" name="lineEdit_recherche">
                   <property name="styleSheet">
                    <string notr="true">QLineEdit
{

background-color: rgb(255,255,255);
color:rgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDateEdit" name="dateEdit">
                   <property name="styleSheet">
                    <string notr="true">QDateEdit
{
color:rgb(0,0,0);
}</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_recherche">
                   <property name="styleSheet">
                    <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                   </property>
                   <property name="text">
                    <string>Recherche</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_stats">
                   <property name="styleSheet">
                    <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                   </property>
                   <property name="text">
                    <string>Afficher tout</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="styleSheet">
                    <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                   </property>
                   <property name="text">
                    <string>Trier</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_2">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <item>
                    <property name="text">
                     <string>ASC</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>desc</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Sexe</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QRadioButton" name="radioButton">
                    <property name="text">
                     <string>Homme</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QRadioButton" name="radioButton_2">
                      <property name="text">
                       <string>Femme</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Filtre</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <property name="styleSheet">
                  <string notr="true">QComboBox
{
background-color: rgb(0,0,0);
}</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>CIN</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sexe</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Date Naissance</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Combinaison</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Nom</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QTableView" name="tableView">
                 <property name="styleSheet">
                  <string notr="true">color:tgb(0,0,0);</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <item>
                    <widget class="QPushButton" name="pushButton_supprimer">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Supprimer</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_modifier">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Modifier</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_details">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Exporter</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_2">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Stats</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_notifier">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Imprimer</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_envoyer">
                     <property name="styleSheet">
                      <string notr="true">QPushButton:!hover{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}

QPushButton:hover{
background-color:rgb(107, 107, 107);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}
QPushButton:click{
background-color:rgb(90, 90, 90);
font: 8pt &quot;MS Shell Dlg 2&quot;;

}</string>
                     </property>
                     <property name="text">
                      <string>Envoyer</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tables</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
